scipy/stats/_multivariate.py

```diff
@@ -3860,7 +3860,7 @@ def rvs(self, dim, size=1, random_state=None):
 shape : array_like, optional
     Positive semidefinite matrix of the distribution. (default ``1``)
 df : float, optional
-    Degrees of freedom of the distribution; must be greater than zero or ``np.inf``. (default ``1``) 
+    Degrees of freedom of the distribution; must be greater than zero. If ``np.inf`` then results are multivariate normal. The default is ``1``. 
```

Comment:
hmm, these should've been on new lines. 
I think we have a char limit check in CI of 79 that fails builds.

Reply:
Okay, fixed.